main.go

```diff
@@ -196,6 +196,7 @@ func wrappedMain() int {
 			Reader:      os.Stdin,
 			Writer:      os.Stdout,
 			ErrorWriter: os.Stdout,
+			PB:          &packer.UiProgressBar{},
```

Comment:
NIT: here we could assign the no-op progress bar and then only during specific cases give the `UiProgressBar` to avoid surprises.

Reply:
oh yeah, inverting it is a little cleaner. That way we'll only use the Progress bar if there's a TTY.